core/cmd/remote_client_test.go

```diff
@@ -259,7 +259,6 @@ func TestClient_CreateExternalInitiator_Errors(t *testing.T) {
 		args []string
 	}{
 		{"no arguments", []string{}},
-		{"not enough arguments", []string{"bitcoin"}},
```

Comment:
Can we move this test case up to the successful `CreateExternalInitiator` tests area? Ideally we'd have a test in the cmd package for this.